doc/contributing/building_ruby.md

```diff
@@ -96,11 +96,20 @@
 5. Build Ruby:
 
     ``` shell
-    make install
+    make
     ```
 
 6. [Run tests](testing_ruby.md) to confirm your build succeeded.
 
+7. Install Ruby:
+
+    ``` shell
+    make install
+    ```
+
+    - If you need to run `make install` with `sudo` and want to avoid document generation with different permissions, you can use
+    `make docs` before `make install` to generate the documentation.
```

Comment:
`make SUDO=sudo install` would not generate not-owned files.

Reply:
I don't know that!